api/types/src/move_types.rs

```diff
@@ -781,6 +785,9 @@ impl From<&AbilitySet> for MoveFunctionGenericTypeParam {
 #[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
 pub struct MoveModuleBytecode {
     pub bytecode: HexEncodedBytes,
+    // We don't need deserialize MoveModule as it should be serialized
+    // from `bytecode`.
+    #[serde(skip_deserializing)]
```

Comment:
Fixing MoveType deserializer requires significant work, because we need to create a new parser to parse references and generic parameters Move type string.
As we should not deserialize `abi` at all, skip deserializing is better option to fix the problem in this PR.